react-native.config.js

```diff
@@ -0,0 +1,61 @@
+const cosmiconfig = require('cosmiconfig');
+const fs = require('fs/promises');
+const path = require('path');
+const pc = require('picocolors');
+
+// TODO: Improve wording (description, errors)
+
+module.exports = {
+  commands: [
+    {
+      name: 'setup-ios-permissions',
+      description: 'No description',
+      func: async () => {
+        const explorer = await cosmiconfig('iosPermissions');
+        const result = await explorer.search();
+
+        if (!result) {
+          console.log(pc.red('No config detected'));
+          process.exit(1);
+        }
+
+        const {config} = result;
+
+        if (!Array.isArray(config)) {
+          console.log(pc.red('Invalid config'));
+          process.exit(1);
+        }
+
+        if (config.length === 0) {
+          console.log(pc.yellow('Empty config'));
+        }
+
+        const iosDirPath = path.join(__dirname, 'ios');
```

Comment:
This path can be different in monorepos and exotic setup. RN cli plug-ins have access to config object part of fn arguments) where you can get "config.project.ios.sourceDir"

Reply:
@thymikee I know, but this is not the project `ios` directory here, but `node_modules/react-native-permissions/ios` 🙂

Reply:
Ugh, I didn't realize when checking on a mobile 😅